module/support/NUbugger/src/Localisation.cpp

```diff
@@ -39,105 +39,19 @@ namespace support {
     using message::localisation::Model;
 
     void NUbugger::provideLocalisation() {
+
         handles["localisation"].push_back(
-            on<Every<100, std::chrono::milliseconds>,
-               Optional<With<std::vector<Ball>>>,
-               Optional<With<std::vector<Self>>>,
+            on<Every<10, Per<std::chrono::seconds>>,
```

Comment:
Change this to trigger functions. `on<Trigger<Self>>` and `on<Trigger<Ball>>`.
Having them grouped like this is pointless and won't update as fast as it could.